### PR TITLE
Chef - Continue instead of breaking on build error

### PR DESCRIPTION
**Pull request by cpagravel**
#### Problem
* While building for Chef CD, if there is a single broken platform, then it could prevent buildable platforms from being built.

#### Change overview
* Use continue instead of break to get as many viable builds as possible.

#### Testing
Tested locally by building with broken platform code with and without the `--keep_going` flag.

### CODE REVIEW

File: examples/chef/chef.py [hunk 1]
```diff
@@ -387,13 +387,13 @@ def main(argv: Sequence[str]) -> None:
                 except RuntimeError as build_fail_error:
                     failed_builds.append((device_name, platform, "build"))
                     flush_print(str(build_fail_error))
-                    break
+                    continue
```
Comment (msandstedt) [inline examples/chef/chef.py:390]: Won't this just increase the amount of time for CI to report build breakages?

**Review by andy31415 — COMMENTED** [review]
```
Chef builds all should continue trying other platform builds in the loop instead of breaking.
```

Why?

@cpagravel

**Comment by andy31415** [comment]
Testing is needed, at a minimum compile with and without the change locally to validate functionality. Eventually introduce an error in the build logic to see if things work as expected.

**Review by woody-apple — APPROVED** [review]
Fast tracking test updates

**Review by andy31415 — APPROVED** [review]

**Pull request merged**